presto-hive/src/test/java/com/facebook/presto/hive/TestHiveHistoryBasedStatsTracking.java

```diff
@@ -179,6 +179,9 @@ public void testPartialAggStatistics()
 
             assertTrue(PlanNodeSearcher.searchFrom(plan.getRoot())
                     .where(node -> node instanceof AggregationNode && ((AggregationNode) node).getStep() == AggregationNode.Step.PARTIAL).findAll().isEmpty());
+
+            String queryGBPartitionKey = "SELECT ds FROM test_orders group by ds";
```

Comment:
Why partition key is special? For a non partition key, for example, `select orderpriority from test_orders group by  orderpriority`, does it have the same problem?